src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -39,16 +35,6 @@ public void execute() {
         rotation = rotationSupplier.getAsDouble();
         
         drivetrainSubsystem.drive(x, y, rotation);
-        
-        if (rotation == 0 && (x != 0 || y != 0)) {
-            if (updateRotationTarget) {
-                drivetrainSubsystem.setRotationTarget(rotationTarget);
-                updateRotationTarget = false;
-            }
-        } else {
-            drivetrainSubsystem.setRotationTarget(null);
-            updateRotationTarget = true;
-        }
```

Comment:
I like the idea of just setting the rotation target to null when rotation is non-zero , this would essentially cancel the rotation target.

It feels like this would be intuitive as if you are driving and set the target but need to evade a defender you want to be able to control it and not have the robot automatically rotate back.